website/content/docs/k8s/connect/cluster-peering/usage/create-sameness-groups.mdx

```diff
@@ -182,42 +179,44 @@ spec:
 apiVersion: consul.hashicorp.com/v1alpha1
 Kind: ExportedServices
 metadata:
-  name: group-a-export
+  name: product-group-export
 spec:
   services:
     - name: api
       namespace: store
       consumers:
-        - samenessGroup: sameness-group-a
+        - samenessGroup: product-group
 ```
 
 </CodeBlockConfig>
 
 </CodeTabs>
 
-For more information about exporting services, including examples of configuration entries that export multiple services at the same time, refer to the [exported services configuration entry reference](/consul/docs/connect/config-entries/exported-services).
+For more information about exporting services, including examples of CRDs that export multiple services at the same time, refer to the [exported services configuration entry reference](/consul/docs/connect/config-entries/exported-services).
 
 After you create each exported services configuration entry, apply it to the Consul server with the following CLI command:
 
 ```shell-session
-$ kubectl apply --filename group-a-export.yaml
+$ kubectl apply -f product-group-export.yaml
 ```
 
-#### Export services for cluster peers and sameness groups at the same time
+#### Export services for cluster peers and sameness groups as part of the same workflow
 
-It is possible to combine the exported services configuration entry for creating sameness groups with the entry required to [export services when establishing cluster peering connections](/consul/docs/k8s/connect/cluster-peering/usage/establish-peering#export-services-between-clusters).
+If you separately follow the steps in this documentation to create a cluster peering connection between two partitions and then add those partitions to a sameness group, you write and apply two separate exported services CRDs: one to export services to the peer, and a second to export services to other members of the group.
 
-In this workflow, configure the `services[].consumers` block with the `samenessGroup` field instead of the `peer` field. You should write the `sameness-group` configuration entry to Consul before referencing it in the `exported-services` configuration entry.
+If you are peering clusters with the goal of creating a sameness group, it is possible to combine these workflows by using a single exported services CRD.
```

Comment:
Just flagging this section for you to update if you want to implement the feedback on the VM version.